### ISSUE TITLE
Use more recent version of humanize?

### ISSUE THREAD

**Issue by velle** [metwork-framework/jinja2_humanize_extension#7]
jinja2 humanize does not work with values that are timezone aware. In order to get the right result, one must do stuff like:

`{{ t.astimezone().replace(tzinfo=None) | humanize_naturaltime }}`

or

`{{ t | humanize_naturaltime(when=datetime.datetime.now(tz=dateutil.tz.gettz()) ) }}`

More recent versions of humanize work well with time/datetime/date objects with timezone set, and that would allow the following to work:

`{{ t | humanize_naturaltime }}`

Are there any specific reasons why the requirements.txt file is set to not use above version 3 of humanize?

**Comment by thebaptiste** [comment]
According to my notes, the reason is that we are using functions which are deprecated with humanize 4.0.
I definitely have to work on it.

**Comment by thebaptiste** [comment]
The deprecated functions are `abs_timedelta` and `date_and_delta`.
These functions are now private (`_abs_timedelta` and `_date_and_delta`), we should probably remove them.

**Comment by thebaptiste** [comment]
I've just released 0.3.0 for compatibily with humanize >= 3.14 (and also 4.0)
Let me know if it's ok for you.
I kept `abs_timedelta` and `date_and_delta` for compatibility (using private humanize functions `_abs_timedelta` and `_date_and_delta`)

**Comment by thebaptiste** [comment]
Release 0.4.0 with all missing humanize missing functions is now available.
I close this issue. 
@velle Please reopen if it's not ok for you.

**Comment by velle** [comment]
@thebaptiste Appologies for the silence. 

I realized that humanize actually has some issues / short comings with how it handles timezone aware values. And I tried to fix those issues, and that was going to be my first code contribution to an open source project. However, I ran into various challenges that I was not able to solve in an elegant way and my effort stalled. 

The reason (though not a good reason) I did not answer in this thread was because I was expecting that my contribution to humanize was around the corner, and I wanted to finish that first so this extension would use the most recent humanize with those fixes. 

Thanks for making those corrections. And thanks for your friendly and helpful communication here. 

I may write again, if/when I succeed with my fixes to humanize.  Sincerely.
